docs/source/development/style-guide/requirement-and-issue-tracking.rst

```diff
@@ -0,0 +1,120 @@
+.. _requirement-and-issue-tracking:
```

Comment:
This document is great. But, I don't think it belongs under the style guide. I think it should go under a new subfolder - maybe `development/project-management`? or just `source/project-management`, for all github issues documentation and such? Actually, the PR document should probably go under that folder as well... what do you think?

Reply:
Yeah, that makes sense. I'll create a project management subfolder. I'm planning to add a page for a SIT timeline with mini-milestones to prepare for each SIT, which could also make sense under project management.